lib/WatcherProcess.php

```diff
@@ -248,10 +249,11 @@ private function onDeadIpcClient(string $readWatcherId, $ipcClient) {
     private function generateWorkerCommand(Console $console): string {
         $parts[] = \PHP_BINARY;
         if ($ini = \get_cfg_var("cfg_file_path")) {
-            $parts[] = "-c \"{$ini}\"";
+            $ini = escapeshellarg($ini);
+            $parts[] = "-c {$ini}";
         }
         $parts[] = "-d zend.assertions=" . ini_get("zend.assertions");
-        $parts[] = __DIR__ . "/../bin/aerys-worker";
+        $parts[] = escapeshellarg(__DIR__ . "/../bin/aerys-worker");
         $parts[] = "-i {$this->ipcServerUri}";
```

Comment:
Shouldn't be a problem, but may be a problem when switched to unix domain sockets and spaces in file paths.